app/src/main/java/com/kickstarter/ui/viewholders/KSViewHolder.java

```diff
@@ -34,7 +34,7 @@ public KSViewHolder(final @NonNull View view) {
    */
   @Override
   public void onClick(final @NonNull View view) {
-    Timber.d("Default KSViewHolder onClick event");
```

Comment:
this shouldn't change since it's describing the default value, i.e. not every VH onClick will be a project click